README.md

```diff
@@ -9,15 +9,45 @@ Homebrew tap for osrf simulation software
 
 To use:
 
-brew tap osrf/simulation
+    brew tap osrf/simulation
+    brew install ignition-citadel
 
 ## To build bottles
 
 The https://build.osrfoundation.org jenkins instance is used for building bottles with the following job:
 
 * [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=generic-release-homebrew_bottle_builder)](https://build.osrfoundation.org/job/generic-release-homebrew_bottle_builder/) https://build.osrfoundation.org/job/generic-release-homebrew_bottle_builder/
 
+This jenkins job takes a `PULL_REQUEST_URL` as an input parameter and uses the
+[homebrew_bottle_creation.bash](https://github.com/ignition-tooling/release-tools/blob/master/jenkins-scripts/lib/homebrew_bottle_creation.bash)
+script from [ignition-tooling/release-tools](https://github.com/ignition-tooling/release-tools),
+which invokes [brew test-bot](https://github.com/Homebrew/homebrew-test-bot)
+with the [following command](https://github.com/ignition-tooling/release-tools/blob/d6417a5c1be87238f155683e2ef70b2e784eb31c/jenkins-scripts/lib/homebrew_bottle_creation.bash#L38-L40):
+
+~~~
+brew test-bot --tap=osrf/simulation \
+  --root-url=https://osrf-distributions.s3.amazonaws.com/bottles-simulation \
+  --ci-pr ${PULL_REQUEST_URL}
+~~~
+
 If you are using the `release.py` script from [osrf/release-tools](https://bitbucket.org/osrf/release-tools),
 a Jenkins bottle builder job will be started automatically.
 Otherwise, create a pull request to this repository and paste the pull request url into the `PULL_REQUEST_URL`
 build parameter.
+
+The job is not configured to run automatically without the `release.py` script
```

Comment:
It's not clear to me what other automatic ways you're referring to here. I remember from past conversation that you talked about triggering the bottle builders when PRs are opened to bump revisions. Is that what this is referring to? I think this could be clarified a bit more.

Reply:
In homebrew-core, they run bottle builds for every commit and upload them to binary, where they remain private and unpublished for 7 days, so if someone submits a malicious pull request it doesn't directly lead to a public result. Our script uploads to s3 directly after a successful build, so we are more careful about starting builds rather than taking a review step after upload.